go/compress.go

```diff
@@ -153,7 +153,9 @@ func decompressExist(exist *CompressedExistenceProof, lookup []*InnerOp) *Existe
 		Path:  make([]*InnerOp, len(exist.Path)),
 	}
 	for i, step := range exist.Path {
-		res.Path[i] = lookup[step]
+		if int(step) < len(lookup) {
```

Comment:
Why are we just ignorign this case rather than returning nil or error?

Reply:
I don't know what it's best. I choose this, but I could equally do something like:

```go
for i, step := range exist.Path {
  if int(step) >= len(lookup) {
    // return error or panic?
  }
  res.Path[i] = lookup[step]
}
```

What do you think should be the best approach?